connector/src/main/java/org/geysermc/connector/network/translators/world/block/BlockTranslator.java

```diff
@@ -97,6 +98,14 @@
         } catch (Exception e) {
             throw new AssertionError("Unable to load Java block mappings", e);
         }
+
+        // Load Block Overrides
+        JsonNode blocksOverride = null;
+        try (InputStream is = Toolbox.getResource("overrides/blocks.json")) {
+            blocksOverride = Toolbox.JSON_MAPPER.readTree(is);
+        } catch (IOException ignored) {
+        }
```

Comment:
This should be cleaned up to be `} catch (IOException ignored) { }`

Reply:
It's a trivial change, but I personally don't think it's required.